src/project.cpp

```diff
@@ -193,94 +170,97 @@ void Project::SetImagesPath(QString images_path) {
 
 bool Project::WriteConfigurationFile() {
   QFile configFile(GetConfigurationFilePath());
-  if (configFile.open(QIODevice::ReadWrite)) {
-    QTextStream stream(&configFile);
-    stream << "PROJECT_CONFIG_VERSION v1.0" << endl;
-    stream << "PROJECT_NAME " << project_name_ << endl;
-    stream << "IMAGES_LOCATION " << GetImagesPath() << endl;
-    stream << "NUMBER_OF_IMAGES " << storage_->NumberOfImages() << endl;
-    for (auto image_path : storage_->GetImages()) {
-      stream << image_path << endl;
-    }
-    stream << "OUTPUT_LOCATION "
-           << GetDefaultOutputPath() << endl;
-  } else {
-    return false;
+  if (!configFile.open(QIODevice::ReadWrite)) { return false; }
+
+  QTextStream stream(&configFile);
+  stream << "PROJECT_CONFIG_VERSION v1.1" << endl;
+  stream << "PROJECT_NAME " << project_name_ << endl;
+  stream << "STATUS " << storage_->GetReconstructionStatus() << endl;
+  stream << "IMAGES_LOCATION " << GetImagesPath() << endl;
+  stream << "NUMBER_OF_IMAGES " << storage_->NumberOfImages() << endl;
+  for (auto image_path : storage_->GetImages()) {
+    stream << image_path << endl;
   }
+  stream << "OUTPUT_LOCATION " << GetDefaultOutputPath() << endl;
+
   configFile.close();
   return true;
 }
 
 bool Project::ReadConfigurationFile() {
   QFile configFile(GetConfigurationFilePath());
 
-  if (!configFile.open(QIODevice::ReadOnly)) {
-    return false;
-  }
+  if (!configFile.open(QIODevice::ReadOnly)) { return false; }
 
   QTextStream stream(&configFile);
   QString temp_line;
 
   temp_line = stream.readLine();
-  if (temp_line != "PROJECT_CONFIG_VERSION v1.0") {
-    std::cerr << "Reading config failed: wrong file version." << std::endl;
+  if (temp_line != "PROJECT_CONFIG_VERSION v1.1") {
```

Comment:
we should somehow react on deprecated config files.
e.g. `if v1.0 { LOG(ERROR) << "Deprecated config file"; }`